### PR TITLE
Document how to enable darker instead of black in vscode.

### LINKED ISSUE #27: Document how to set the `--diff` flag for VS Code
_Originally posted by @Carreau in https://github.com/akaihola/darker/issues/22#issuecomment-656703323_

### PR DESCRIPTION
**Pull request by Carreau**
Closes #27

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -225,10 +229,14 @@ Visual Studio Code
...
-3. Add these configuration options::
+3. Add these configuration options to VS code, ``Cmd-Shift-P``, ``Open Settings (JSON)``::
+   
 
     "python.formatting.provider": "black",
-    "python.formatting.blackPath": "<install_location_from_step_2>"
+    "python.formatting.blackPath": "<install_location_from_step_2>",
+    "python.formatting.blackArgs": ["--diff"],
+
+Extra argument can be passed to ``darker`` along side, but ``--diff`` is necessary.
```
Comment (akaihola) [inline README.rst:239]: The meaning of this wasn't immediately obvious to me after reading it once. Would this wording with an example included be clear:
```suggestion
You can pass additional arguments to ``darker`` in the ``blackArgs`` option (e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```

File: README.rst [hunk 2]
```diff
@@ -122,12 +122,16 @@ The following `command line arguments`_ can also be used to modify the defaults:
      -l LINE_LENGTH, --line-length LINE_LENGTH
                            How many characters per line to allow [default: 88]
 
+     --diff                Don't write the files back, just output a diff for each file on stdout
+
```
Comment (akaihola) [inline README.rst:126]: I would either use a blank line between every option, or remove the blank line from here, too.

Maybe also keep the same ordering as in the output from `darker --help`, and if it's not logical, fix it in the code as well?

```suggestion
     --diff                Don't write the files back, just output a diff for each file on stdout

```

File: README.rst (hunk 1 above)
Comment (Carreau) [reply README.rst]: sounds good.

File: README.rst (hunk 2 above)
Comment (Carreau) [reply README.rst]: ok.

File: README.rst [hunk 3]
```diff
@@ -112,9 +112,15 @@ The following `command line arguments`_ can also be used to modify the defaults:
...
 
+     --diff                Don't write the files back, just output a diff for
+                           each file on stdout
+
      --check               Don't write the files back, just return the status.
                            Return code 0 means nothing would change. Return code
                            1 means some files would be reformatted.
+
+     -i, --isort           Also sort imports using the `isort` package
+
```
Comment (Carreau) [inline README.rst:123]: Reordered to match `--help` and added the `-i, --isort` option

File: README.rst [hunk 4]
```diff
@@ -236,7 +235,7 @@ Visual Studio Code
     "python.formatting.blackPath": "<install_location_from_step_2>",
     "python.formatting.blackArgs": ["--diff"],
 
-Extra argument can be passed to ``darker`` along side, but ``--diff`` is necessary.
+You can pass additional arguments to ``darker`` in the ``blackArgs`` option (e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```
Comment (akaihola) [inline README.rst:243]: Actually, could we keep the README file a bit narrower for the benefit of people reading it unrendered? Sorry for not noticing this in my original suggestion.

```suggestion
You can pass additional arguments to ``darker`` in the ``blackArgs`` option
(e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```

Comment (Carreau) [reply README.rst]: Oh, yeah did not realize either.

File: README.rst [hunk 5]
```diff
@@ -112,9 +112,15 @@ The following `command line arguments`_ can also be used to modify the defaults:
 
 .. code-block:: shell
 
+     --diff                Don't write the files back, just output a diff for
+                           each file on stdout
+
```
Comment (Carreau) [inline README.rst:117]: TODO: update this if #46 is merged first.

**Pull request merged**